packages/teams/src/types.ts

```diff
@@ -43,6 +48,95 @@ export interface IPrivPropValues {
   value: string;
 }
 
+/**
+ * Team status interface, comes from ember application
+ * status for an individual team
+ */
+export interface ITeamStatus {
+  id: string;
+  isOk: boolean;
+  isMissing: boolean;
+  isBroken: boolean;
+  canFix: boolean;
+  isMember: boolean;
+}
+
+/**
+ * Teams status interface, comes from ember application
+ * Comprised of status of core/content/followers teams
+ */
+export interface ITeamsStatus {
+  core: ITeamStatus;
+  content: ITeamStatus;
+  followers: ITeamStatus;
+}
+
+/**
+ * User object returned from add users modal in ember application
+ * It extends the IUser interface with an additional property
+ * that denotes what org relationship the user might have (world|org|community)
+ */
+export interface IUserModalObject extends IUser {
```

Comment:
The name of this interface is puzzling... I get that it's returned from the Modal, but what is it actually adding? also you can type this like
```
orgRelationship:  'world' | 'org' | 'community'
```

Reply:
if it's optional keep the `?`